src/store.js

```diff
@@ -169,6 +170,7 @@ Store.prototype.deselect = function(featureIds, options = {}) {
     if (!this._selectedFeatureIds.has(id)) return;
     this._selectedFeatureIds.delete(id);
     this._changedFeatureIds.add(id);
+    this._selectedCoordinates = this._selectedCoordinates.filter(point => point.feature_id !== id);
```

Comment:
This will filter out all points that don't match all selected features. In our current use case, only one feature is selected when coordinates are selected, but if we change that this will create unexpected behaviors.

Let's file a clean up issue for this as its ok for now.